conftest.py

```diff
@@ -26,12 +26,15 @@ def __init__(self, **kwargs):
 @pytest.fixture(scope='session')
 def cloud_config():
     """Provides a configuration object as a proxy to environment variables."""
+    speech_storage_uri = 'gs://' + os.environ.get('CLOUD_STORAGE_BUCKET')
```

Comment:
@ryanmats there's already the `storage_bucket` thing here, there isn't any need to add another one.